### PR TITLE
#21 Fixed Cautery rotation

### PR DESCRIPTION
**Pull request by christinayan**
Updated CauteryModelToCauteryTip transform by rotating it to correct the cautery's orientation.

### CODE REVIEW

**Comment by lassoan** [comment]
Thank you. How did you determine this transform?

**Comment by christinayan** [comment]
I got the transform matrix by adjusting the rotation sliders in the Transforms module. Tamas and I tested it again today, the cautery looks better but still needs some adjustment. I am working on this and will make another commit asap.

File: LumpNav/LumpNav.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def addValuesToDefaultConfiguration(self):
                    'TipToSurfaceDistanceTrajectory' : 'True',
                    'NeedleModelToNeedleTip' : '0.0 1.0 0.0 0.0 0.0 0.0 1.0 0.0 1.0 0.0 0.0 0.0 0 0 0 1',
                    'NeedleBaseToNeedle' : '1 0 0 20.93 0 1 0 14.00 0 0 1 -4.27 0 0 0 1',
-                   'CauteryModelToCauteryTip' : '0 0 1 0 0 -1 0 0 1 0 0 0 0 0 0 1',
+                   'CauteryModelToCauteryTip' : '0.03 0.03 1.01 0.00 -0.23 0.98 -0.02 0.00 -0.97 -0.23 0.03 0.00 0 0 0 1',
```
Comment (lassoan) [inline LumpNav/LumpNav.py]: Do you know why you have value >1.0 in the rotation part of the matrix?
(it's not much, only 1.01, so probably the magnification is not visible, I'm just wondering what could have happened)

Comment (christinayan) [inline LumpNav/LumpNav.py]: I got the transform matrix by adjusting the sliders in the transforms module to fix the orientation of the cautery.

Comment (lassoan) [inline LumpNav/LumpNav.py]: Ok, thank you, it's good to know that the transforms module has this slight numerical imprecision.

**Pull request merged**